package.json

```diff
@@ -77,7 +77,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "CI=true yarn test && lint-staged"
+      "pre-commit": "CI=true yarn test && (cd /functions && npm test) && lint-staged"
```

Comment:
we're using `yarn` for the frontend and `npm` for the backend? maybe we can go for one and unify both folders

Reply:
This was corrected with my commit.